recipe/meta.yaml

```
@@ -1,46 +1,59 @@
 {% set name = "tslearn" %}
-{% set version = "0.5.3.2" %}
-{% set sha256 = "acf4015b07207ad6a645ddcc8eec04d8431716b24db66a745ab29fa22f772820" %}
+{% set version = "0.6.3" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: c855c0b784b42dc4c50f4eaf524ec7ed658037e86fc14af38ca3b66c75dd6adb
 
 build:
   number: 0
-  skip: true  # [py2k]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py>312 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - {{ compiler('c') }}
   host:
     - python
     - pip
-    - cython
-    - numpy
+    - numpy 2.0 # [py<313]
+    - numpy 2.1 # [py>=313]
+    - setuptools
+    - wheel
   run:
     - python
-    - {{ pin_compatible('numpy') }}
+    - numpy
     - scipy
     - scikit-learn
     - numba
     - joblib
```

```suggestion
    - joblib
    - tensorflow >=2
```
I think the tests were failing because tensorflow is actually needed for one of their modules. See [Changelog](https://github.com/tslearn-team/tslearn/blob/v0.6.3/CHANGELOG.md#v040)

---

I originally had `tensorflow` in the tests, but ended up skipping due to the following error:

```
 ModuleNotFoundError: No module named 'tensorflow.keras'
 ```

---

Ahhh, I see. Probably because `tensorflow` and `keras` are coupled so they both need to be added. I think they should be added to the run or run_constrained section.

---

That's correct. tensorflow >=2 is a runtime dependency. The maintainers forgot to add it to setup.py.
@psteyer Could to extend test.imports of the recipe by including all python modules from https://github.com/tslearn-team/tslearn/tree/v0.6.3/tslearn? This way we'll be confident that all imports don't fail